src/PIL/Jpeg2KImagePlugin.py

```diff
@@ -200,13 +200,21 @@ def _open(self):
                 "jpeg2k",
                 (0, 0) + self.size,
                 0,
-                (self.codec, self.reduce, self.layers, fd, length),
+                (self.codec, self._reduce, self.layers, fd, length),
             )
         ]
 
+    @property
+    def reduce(self):
+        return self._reduce or super().reduce
```

Comment:
Probably some comment with a link to the issue is required.

Reply:
Okay, done